README.md

```diff
@@ -34,6 +35,7 @@ This is the temporary home for standards and guidance relating to software devel
   - [Security standards](/standards/security_standards.md)
   - [TSQL and SQL Server database standards](/standards/tsql_and_sqldb_standards.md)
   - [Version control standards](/standards/version_control_standards.md)
+  - [Docker standards](/standards/docker_standards.md)
```

Comment:
This should be in the list alphabetically, should be "Container standards" and should link to the container_standards.md file.

Reply:
Done